### ISSUE TITLE
GUI BUG: Add Instance drop-down menus break when display config changes

### ISSUE THREAD

**Issue by MrLazerface** [bitfocus/companion#362]
When companion is running, making any display changes to the host PC will then cause the drop down selection boxes to not scale/scroll properly. It doubles up the list when selecting 'Add by Category' and won't scroll properly when selecting 'Add by Manufacturer'. This will occasionally happen when changing display scaling in windows 10 as well.

1. Launch Companion
2. Connect another display and/or change display scaling in your system
3. Open companion config and try to add an instance
4. See GUI issues

**Expected behavior**
Shouldn't do that. Problem may exist with windows 10 display scaling/display handling.

**Screenshots**
2 attached screenshots.
![broken config page 2](https://user-images.githubusercontent.com/40131644/49046346-10d8dd80-f191-11e8-88c1-56bea76ce183.png)
![broken config page 1](https://user-images.githubusercontent.com/40131644/49046347-10d8dd80-f191-11e8-9475-850675771fbc.png)

**Desktop (please complete the following information):**
 - Windows 10 Home (x64)
 - Chrome 70.0.3538.110
 - Companion 1.2.0 (but I recall it going back thru the 1.1.1 builds)

**Additional context**
Restarting companion fixes the issue.

**Comment by willosof** [comment]
It's 50% your browser and 50% companion (me). But, I didn't see this as a lanuch blocker for 1.2.0 due to the search field below.

**Comment by MrLazerface** [comment]
@willosof Agreed, not a deal breaker now that search has been added. But still a bug that should be looked at eventually (more features and modules first!).

**Comment by Hitower78** [comment]
Hi
In Version 1.3 and the Firefox 64.0.2 is the same Problem: 
The Dropdown List is doubled and perhaps the manufactorlist isnt show complete
The search function is fine, thx 👍 

I have not change the Displaysetting (Dell XPS15 HD Display), my list end at Rossvideo

**Comment by MeestorX** [comment]
Just a friendly bump - you can't scroll the list and it goes off the screen, as noted.

Not an easy fix I take it?

**Comment by dgw** [comment]
Freshly installed (I've never used Companion before) on Win10 1909 x64, using Chrome 85. The page scrolls behind the "Add" drop-downs; can't see the full list of categories or manufacturers.

![image](https://user-images.githubusercontent.com/164140/92988806-74321d80-f494-11ea-92e1-adc134bee3d8.png)

Would love to explore what this can do. Workaround from the OP (restarting Companion) does not help. Issue is present right from startup; no need to change anything.

**Comment by amoyer1974** [comment]
Even in Chrome (Version 87.0.4280.88) I am unable to see the modules from the categories or manufacturers at the bottom of the list.

**Comment by PhotoJoseph** [comment]
I'll bump that while the ability to search may negate the need for users knowing what they're looking for, as a new user, I'm using this list to see what devices I may have that I haven't even considered adding might be supported. Scrolling the list is important for discovery.

**Comment by Julusian** [comment]
I have some ideas on how to reimplement this to be better that I shall explore in the ui overhaul I am working on

**Comment by josephdadams** [comment]
@Julusian new react ui fixes this?

**Comment by llamafilm** [comment]
Why is this no longer relevant?  I'm new to Companion so I'm not sure what "new react ui" means.  I just installed version 2.1.3 on macOS 11.6 and I can't scroll the manufacturer list.  Same problem in Chrome 96 and Safari 15.

**Comment by josephdadams** [comment]
Because it’s fixed in v2.2 which is a beta release but the one everyone uses anyway.
